doc/ompl_interface/ompl_interface_tutorial.rst

```diff
@@ -71,3 +96,14 @@ The configuration of these optimization objectives can be done in the *ompl_plan
 
 For more information on the OMPL optimal planners, the reader is referred to the
 `OMPL - Optimal Planning documentation <http://ompl.kavrakilab.org/optimalPlanning.html>`_.
+
+Post-Processing Smoothing
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Note: there is a limit to how much smoothing can help reduce indirect routes.
+
+You can adjust the amount of time MoveIt! spends on smoothing by increasing the planning time. Any remaining time after an initial plan is found, but before the ``allowed_planning_time`` is exhausted, will be used for smoothing.
```

Comment:
I believe that MoveIt already does path hybridization (taking the best parts of N different RRT runs and splicing them together) [here](https://github.com/ros-planning/moveit/blob/ac7862b1688488e8867dfa4068ed6d9986dff7ce/moveit_planners/ompl/ompl_interface/src/model_based_planning_context.cpp#L676).  Might be worth it to mention that changing [num_planning_attempts](https://github.com/ros-planning/moveit_msgs/blob/53496e71126d31137d60f573d852126da9e1346d/msg/MotionPlanRequest.msg#L32) will affect the quality as well.